src/metaTypes.h

```diff
@@ -10,6 +10,7 @@
   See the License for more information.
 ============================================================================*/
 #include "localMetaConfiguration.h"
+#include <stdint.h>
```

Comment:
I believe this header is first available with VS9 SP1. There have been some portability issue with this header. I still carry around a copy in SimpleITK to help with compatibility.